tiptapy/templates/dynamic_table.html

```
@@ -1,14 +1,20 @@
 <table>
     {%- set visible_columns = [] -%}
+    {%- if node.attrs.columns and node.attrs.columns|length > 0 -%}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)